sys/include/net/gnrc/gomach/gomach.h

```diff
@@ -7,10 +7,25 @@
  */
 
 /**
- * @defgroup    net_gnrc_gomach A traffic-adaptive multi-channel MAC
+ * @defgroup    net_gnrc_gomach GoMacH
  * @ingroup     net_gnrc
- * @brief       A traffic adaptive MAC protocol that provides high traffic
- *              adaptability, high energy efficiency and high robustness.
+ * @brief       GoMacH is, "a General, nearly Optimal MAC protocol
+ *              for multi-Hop communications", for IEEE 802.15.4 IoT/WSNs networks.
+ *              It is designed to be a traffic adaptive MAC protocol that provides
+ *              high traffic adaptability, high energy efficiency and high robustness.
```

Comment:
@smlng But we have the removed description above texts displayed here~ :-)

Reply:
Ah, you mean the description should be a `one-liner`, i.e., not longer than one line (like 2 or 3 lies)? @smlng @miri64

Reply:
exactly, the text is fine, just add a one-liner for `@brief` and move these 2 sentences after that (2 empty lines) so its part of the longer description.

Reply:
Thanks!~ Learned and worked on it~ :-)